gammapy/modeling/models/core.py

```diff
@@ -431,6 +434,18 @@ def read(cls, filename):
     def from_yaml(cls, yaml_str, path=""):
         """Create from YAML string."""
         data = yaml.safe_load(yaml_str)
+        if "metadata" in data.keys():
+            creator_version = data["metadata"]["creator"].split()[1]
```

Comment:
I think this will cause issues because there is no guarantee that creator metadata will always be structured this way. `split()[1]` can result in unwanted failures if `split()` returns only one element.